random/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/blocky/as-demo
+
+go 1.22.6
+
+require github.com/blocky/basm-go-sdk v0.1.0-beta.4
```

Comment:
revisit if we're ready to update this dependency and maybe the on in shell.nix as well